docker-hub/download-rate-limit.md

```diff
@@ -61,17 +61,17 @@ To get a token with a user account (if you are authenticating your pulls) - don'
 $ TOKEN=$(curl --user 'username:password' "https://auth.docker.io/token?service=registry.docker.io&scope=repository:ratelimitpreview/test:pull" | jq -r .token)
 ```
 
-Then to get the headers showing your limits, run the following (keep in mind that requesting a manifest emulates a pull and will count against the limits):
+Then to get the headers showing your limits, run the following:
 
 ```
-$ curl -v -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep RateLimit
+$ curl -I -H "Authorization: Bearer $TOKEN" https://registry-1.docker.io/v2/ratelimitpreview/test/manifests/latest 2>&1 | grep RateLimit
 ```
 
 Which should return something like this:
 
 ```
-< RateLimit-Limit: 100;w=21600  
-< RateLimit-Remaining: 76;w=21600
+RateLimit-Limit: 100;w=21600
+RateLimit-Remaining: 76;w=21600
```

Comment:
Maybe add a comment after this to be clear that it is expected that if you are authenticated as pro/team/legacy, you won't see these headers (I recall GitHub or CircleCI had to ask us to make sure)